From c2ad9ae674eab8adf3b98dc497b7711a03ff3333 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 15 Feb 2017 18:15:01 -0800
Subject: [PATCH] MORAY-398 moray caught connecting to deposed peer Reviewed
 by: Cody Peter Mello <cody.mello@joyent.com> Approved by: David Pacheco
 <dap@joyent.com>

---
 bin/zkConnTest.js     | 2 +-
 bin/zkGetWatchTest.js | 2 +-
 manatee.js            | 2 +-
 package.json          | 2 +-
 test/client.test.js   | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/bin/zkConnTest.js b/bin/zkConnTest.js
index 8f7e0b3..1837156 100755
--- a/bin/zkConnTest.js
+++ b/bin/zkConnTest.js
@@ -17,7 +17,7 @@
  */
 var bunyan = require('bunyan');
 var once = require('once');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 
 var LOG = bunyan.createLogger({
     level: (process.env.LOG_LEVEL || 'info'),
diff --git a/bin/zkGetWatchTest.js b/bin/zkGetWatchTest.js
index 5f2931c..c8119b4 100755
--- a/bin/zkGetWatchTest.js
+++ b/bin/zkGetWatchTest.js
@@ -17,7 +17,7 @@
  */
 var bunyan = require('bunyan');
 var once = require('once');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 
 var LOG = bunyan.createLogger({
     level: (process.env.LOG_LEVEL || 'info'),
diff --git a/manatee.js b/manatee.js
index 2d2a779..a8ac483 100644
--- a/manatee.js
+++ b/manatee.js
@@ -22,7 +22,7 @@ var util = require('util');
 var uuid = require('uuid');
 var vasync = require('vasync');
 var verror = require('verror');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 
 var EventEmitter = require('events').EventEmitter;
 
diff --git a/package.json b/package.json
index a57206f..5746cca 100644
--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
         "backoff": "1.2.0",
         "bunyan": "0.21.3",
         "node-uuid": "1.3.3",
-        "node-zookeeper-client": "0.2.1",
+        "joyent-zookeeper-client": "0.2.2",
         "once": "1.3.1",
         "pg": "3.0.3",
         "posix-getopt": "1.0.0",
diff --git a/test/client.test.js b/test/client.test.js
index 3d0ffb4..64f2fb1 100644
--- a/test/client.test.js
+++ b/test/client.test.js
@@ -13,7 +13,7 @@ var bunyan = require('bunyan');
 var once = require('once');
 var vasync = require('vasync');
 var manatee = require('../manatee');
-var zkClient = require('node-zookeeper-client');
+var zkClient = require('joyent-zookeeper-client');
 var test = require('tape').test;
 
 var LOG = bunyan.createLogger({
-- 
2.21.0

